androidApp/src/main/java/com/mbta/tid/mbta_app/android/map/HomeMapView.kt

```diff
@@ -336,12 +343,20 @@ fun HomeMapView(
                     layerManager.`object` = MapLayerManager(map.mapboxMap, context)
             }
 
-            if (!viewportProvider.isFollowingPuck && lastNearbyTransitLocation != null) {
-                CircleAnnotation(
-                    point = lastNearbyTransitLocation.toPoint(),
-                    circleColorString = "#ba75c7",
-                    circleRadius = 10.0
-                )
+            if (
+                isNearbyNotFollowing &&
+                    lastNearbyTransitLocation != null &&
```

Comment:
🎱 (stylistic) `hasStableNearbyTransitLocation` would be help me here, I blame Ruby for my dislike of long booleans.